docs/references/javascript/clerk/session-methods.mdx

```diff
@@ -50,16 +50,16 @@ function setActive({
 }: SetActiveParams): Promise<void>;
 ```
 
-An **experimental** method used to set the active session and/or organization.
+An experimental method used to set the active session and/or organization.
 
 > This method will eventually replace `setSession`.
 
 ### `SetActiveParams`
 
 | Name | Type | Description |
 | --- | --- | --- |
-| `session` | `Session` \| `string` \| `null` | The session resource or session id (string version) to be set as active. If `null`, the current session is deleted. |
-| `organization` | [`Organization`](/docs/references/javascript/organization/organization) \| `string` \| `null` | The organization resource or organization id (string version) to be set as active in the current session. If `null`, the currently active organization is removed as active. |
+| `session` | `Session \| string \| null` | The session resource or session ID (string version) to be set as active. If `null`, the current session is deleted. |
+| `organization` | `[Organization](/docs/references/javascript/organization/organization) \| string \| null` | The organization resource or organization ID (string version) to be set as active in the current session. If `null`, the currently active organization is removed as active. |
```

Comment:
The prior syntax was correct, this won't render the link properly as-is.

Reply:
ACK. its not gonna be consistent with how other types are formatted :~(